plugin/trino-mysql/src/main/java/io/trino/plugin/mysql/MySqlClient.java

```diff
@@ -471,6 +486,17 @@ public Optional<ColumnMapping> toColumnMapping(ConnectorSession session, Connect
         return Optional.empty();
     }
 
+    private static LongReadFunction timestampMillisReadFunction()
+    {
+        return (resultSet, columnIndex) -> toTrinoTimestampMillis(resultSet.getObject(columnIndex, LocalDateTime.class));
```

Comment:
you probably should implement isNull as well

https://github.com/trinodb/trino/blob/9c4ef0e98606302dd593072145f9a75d967cfecf/plugin/trino-mysql/src/main/java/io/trino/plugin/mysql/MySqlClient.java#L500-L502